docker-rancher/azuredeploy.json

```diff
@@ -0,0 +1,163 @@
+      "type": "string",
+      "metadata": {
+        "description": "Username for the Virtual Machine."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for the Virtual Machine."
+      }
+    },
+	"deployPrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "The prefix used for the nodes & dns. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+      }
+    },
+	"vmSize": {
+	  "type": "string",
+	  "metadata": {
+	    "description": "The size of the virtual machine used for the deployment"
+	  },
+	  "allowedValues": [
+		"Standard_A0",
+		"Standard_A1",
+		"Standard_A2",
+		"Standard_A3",
+		"Standard_A4",
+		"Basic_A0",
+		"Basic_A1",
+		"Basic_A2",
+		"Basic_A3",
+		"Basic_A4",
+		"Standard_A5",
+		"Standard_A6",
+		"Standard_A7",
+		"Standard_D1",
+		"Standard_D2",
+		"Standard_D3",
+		"Standard_D4",
+		"Standard_D11",
+		"Standard_D12",
+		"Standard_D13",
+		"Standard_D14",
+		"Standard_D1_v2",
+		"Standard_D2_v2",
+		"Standard_D3_v2",
+		"Standard_D4_v2",
+		"Standard_D5_v2",
+		"Standard_D11_v2",
+		"Standard_D12_v2",
+		"Standard_D13_v2",
+		"Standard_D14_v2",
+		"Standard_DS1",
+		"Standard_DS2",
+		"Standard_DS3",
+		"Standard_DS4",
+		"Standard_DS11",
+		"Standard_DS12",
+		"Standard_DS13",
+		"Standard_DS14",
+		"Standard_G1",
+		"Standard_G2",
+		"Standard_G3",
+		"Standard_G4",
+		"Standard_G5",
+		"Standard_GS1",
+		"Standard_GS2",
+		"Standard_GS3",
+		"Standard_GS4",
+		"Standard_GS5",
+		"Standard_A8",
+		"Standard_A9",
+		"Standard_A10",
+		"Standard_A11"
+	  ],
+	  "defaultValue": "Standard_A2"
+	},
+	"deploymentType": {
+      "type": "string",
+      "defaultValue": "Nodes",
+      "allowedValues": [
+        "Nodes",
+        "Server"
+      ],
+      "metadata": {
+        "description": "Choose if you want to add nodes to an existing server or deploy a server without any nodes."
+      }
+    },
+    "nodesApi": {
+	  "type": "string",
+	  "defaultValue": "http://myrancherhost.kvaes.be:8080/v1/scripts/AA12A123A1A1234A12A1:1234567890000:aAaaaaA1AaAaAlaAAaaaaAAAaaA",
+      "metadata": {
+        "description": "(Ignored for deploymentType server) The api link to your RancherHost."
+      }
+    },
+	"nodesCount": {
+	  "type": "int",
+	  "defaultValue": 1,
+      "metadata": {
+        "description": "(Ignored for deploymentType server) The amount of nodes to be provisioned"
+      },
+	  "minValue": 1,
+	  "maxValue": 250
+    },
+	"templateBase": {
+      "type": "string",
+      "defaultValue": "https://raw.githubusercontent.com/azure/azure-quickstart-templates/master",
+      "metadata": {
+        "description": "Change this value to your repo name if deploying from a fork"
+      } 
+    }
+  },
+  "variables": {
+    "apiVersion": "2015-11-01",  
+	"deployTypeParam": {  
+        "Nodes": "azuredeploy.nodes.json",
```

Comment:
It's ok to call the files just nodes.json and server.json. It also reduces the confusion with multiple azuredeploy looking files since there is expected to be one azuredeploy.json file that is the main deployment file.